### ISSUE TITLE
View, center of rotation

### ISSUE THREAD

**Issue by porzioev** [Ultimaker/Cura#13218]
### Is your feature request related to a problem?

1. at any time
2. trying to rotate the view
3. since the pivot point is too far from the center of the scene, the objects and the printing plate splash out of the frame.

### Describe the solution you'd like

solution:
Set the pivot point in the center of the depth of field or object or simply in the center of the build plate. 

### Describe alternatives you've considered

Maybe making the center of rotation selectable from the settings. Or rather by making the center of rotation visible and selectable as in photoshop or cad software.

### Affected users and/or printers

The usability of the program itself.

**Comment by MariMakes** [comment]
Hey @porzioev 

Sorry, it took us a while to get back to you 😞 

Are you aware of the preference Center Camera when Item is Selected?
This seems to be a solution to what you are describing. 
![image](https://user-images.githubusercontent.com/40423138/199242257-4900d090-554a-4e02-aa93-eeccc4c2c4b8.png)

Would you still like to keep this issue open or do you agree that we can close it?

**Comment by porzioev** [comment]
Yes this is a good solution. Also if it would be better to dynamically select the pivot point just by auto detecting from mouse position at the moment of click and drag.
